src/account/ModuleManagerInternals.sol

```diff
@@ -163,17 +158,6 @@ abstract contract ModuleManagerInternals is IModuleManager {
             _setExecutionFunction(selector, isPublic, allowGlobalValidation, module);
         }
 
-        length = manifest.validationFunctions.length;
```

Comment:
Did we get rid of preValidationHooks installation in `installModule` in a previous PR? Don't see them in code anymore.

Reply:
We did, I think in the PR that introduced `installValidation`. Previously, preValidationHooks were associated with selectors, which also associated them to validations because there was a 1:1 mapping between selectors and validation. But when we allowed multiple validation functions, we needed a way to associate the preValidationHooks, so they became `installValidation` only.